django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
ATS should take all parameters into account, because `/some-index?page=1` and `/some-index?page=2` - are different pages and should have different cache. If ATS ignores these parameters it means that configuration is wrong.

Actually we also have a `Vary` header for this functionality - Karl added it later (see `EnquiryFormPage`).